test/tests.transform.js

```diff
@@ -47,3 +47,102 @@ describe("_blockHoist nodes", function() {
     assert.deepEqual(names, ["hoistMe", "doNotHoistMe", "oyez"]);
   });
 });
+
+context("functions", function() {
```

Comment:
hah, thanks. is there anything else i can do to help expedite this merge? it makes a huge difference for me (and probably anyone else using `redux-sagas`).